src/components/HomePage.js

```diff
@@ -1,7 +1,8 @@
 import React, { Component } from 'react';
 // import { connect } from 'react-redux';
 import { Link } from 'react-router';
-import Header from './Header';
+// import Header from './Header';
+import Test from './Test'
```

Comment:
(Part One)

So, it looks like you made two files: `components/Test` and `containers/TestContainer`. The latter connecting the former to Redux. You're in `components/App` so when you import from `./Test`, you're getting `components/Test` (also known as the version that's not at all hooked up to Redux).

So, the first step is to change it to:

```js
import Test from '../containers/TestContainer';
```

This is a similar issue to mjvalade/weather-forecast#9.

This fix however, will blow up a bunch of code. So, let's follow the trail.